it/mail/chrome/communicator/wallet/wallet.properties

```diff
@@ -33,51 +33,51 @@
 # the terms of any one of the MPL, the GPL or the LGPL.
 #
 # ***** END LICENSE BLOCK *****
-IncorrectKey_TryAgain? = Master password incorrect. Do you want to try again?
-WantToCaptureForm? = Do you want to remember the values you filled in?
-WantToSavePasswordObscured? = Password Manager can remember this logon and enter it automatically the next time you return to this website.##Do you want Password Manager to remember this logon?
-WantToSavePasswordEncrypted? = Password Manager can remember this logon and enter it automatically the next time you return to this website.##Do you want Password Manager to remember this logon?
-xWantToSavePasswordObscured? = Do you want to remember your user name and password?##Stored values are not password protected.
-xWantToSavePasswordEncrypted? = Do you want to remember your user name and password?##Stored values are password protected.
+IncorrectKey_TryAgain? = La Password principale non è corretta. Vuoi provare di nuovo?
+WantToCaptureForm? = Vuoi ricordare i valori che hai compilato?
+WantToSavePasswordObscured? = Il Gestore password può ricordare i dati di accesso e inserirli automaticamente quando visiti di nuovo questo sito web.##Vuoi che il Gestore password ricordi questi dati di accesso?
+WantToSavePasswordEncrypted? = Il Gestore password può ricordare i dati di accesso e inserirli automaticamente quando visiti di nuovo questo sito web.##Vuoi che il Gestore password ricordi questi dati di accesso?
+xWantToSavePasswordObscured? = Vuoi ricordare il tuo nome utente e la tua password?##I valori archiviati non sono protetti da password.
+xWantToSavePasswordEncrypted? = Vuoi ricordare il tuo nome utente e la tua password?##I valori archiviati non sono protetti da password.
 #translator's note: # in following line will be converted to linefeed by program
-firstPassword = Una Master Password è utilizzata per proteggere i dati sensibili registrati sul vostro disco fisso.##Le prossime versioni utilizzeranno un algoritmo di encrittazione evoluto.#(Lasciate in bianco i campi per non utilizzare una Master Password).#
-password = Inserite la Master Password.
+firstPassword = Si utilizza una Password principale per proteggere i dati sensibili Single Sign-on archiviati sul tuo disco fisso.##Le prossime versioni utilizzeranno un algoritmo di crittografia evoluto.#(Se non si desidera utilizzare una Password principale non compilare i campi).#
```

Comment:
CORRECTIONS NEEDED? "Single Sign-on" is added from scratch, if compared to the previous version, and maybe it should be removed. Please check the source language.